flake.nix

```diff
@@ -25,7 +25,7 @@
           }: {
             devShells.default = pkgs.mkShell {
               inputsFrom = [ inputs.holochain.devShells.${system}.holonix ];
-              packages = [ pkgs.nodejs-18_x ];
+              packages = [ pkgs.nodejs-18_x pkgs.sqlite ];
```

Comment:
Please could you add a note to take this out in favour of https://github.com/holochain/holochain/pull/2248 which I will backport asap?